builds/fabric8-che/assembly/user-auth-valve/pom.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0"?>
+<!--
+
+    Copyright (c) 2017 Red Hat Inc.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+
+    Contributors:
+      Red Hat Inc. - initial API and implementation
+
+-->
+
+<project
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+    xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>fabric8-ide-assembly-parent</artifactId>
+        <groupId>com.redhat.che</groupId>
+        <version>@redhat.che.version@</version>
+    </parent>
+    <artifactId>user-auth-valve</artifactId>
```

Comment:
I've not preference honestly. For me `UserAuthValve` means that it add s*user authentification*, on top of Keycloak auth, which seems what the issue was about. Could we open a new issue for this and let @amisevsk decide about this and fix it in a new PR if necessary ?